megablocks/layers/dmoe_test.py

```diff
@@ -94,7 +95,7 @@ def testdMoE_Forward(self, bs, sl, hs, num_experts, top_k,
 
         _, _, _, layer = test_modules(
             hidden_size=hs,
-            ffn_hidden_size=hs * 2,
+            ffn_hidden_size=hs,
```

Comment:
why this change?

Reply:
undid this change since it's not needed anymore. Additional test case covers the fix without this change.